src/ripple/app/tx/LocalTxs.h

```diff
@@ -36,7 +36,7 @@ class LocalTxs
     virtual ~LocalTxs () = default;
 
     // Add a new local transaction
-    virtual void push_back (LedgerIndex index, STTx::ref txn) = 0;
+    virtual void push_back (LedgerIndex index, std::shared_ptr<STTx const> const& txn) = 0;
```

Comment:
Yeah, its a long line. But as a general rule unless I am refactoring an entire file, when I do search and replace edits I do not reformat the target file.